src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -190,26 +279,26 @@ export const NotForSale = ({ collection }: { collection: CollectionInfoForAsset
   )
 }
 
+const SubHeader = styled(ThemedText.SubHeader)`
+  color: ${({ theme }) => theme.textPrimary};
+`
+
 export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps) => {
   const { account } = useWeb3React()
-  const cheapestOrder = asset.sellorders && asset.sellorders.length > 0 ? asset.sellorders[0] : undefined
-  const expirationDate = cheapestOrder
-    ? new Date((cheapestOrder as Deprecated_SellOrder).orderClosingDate ?? (cheapestOrder as SellOrder).endAt)
-    : undefined
```

Comment:
expiration isn't showing for me and I believe it's the editing of this logic

Reply:
some orders don't have a closing date.  I forget what shows up but this is intentional